### PR TITLE
[9.0][MIG]account_banking_mandate: Scritp migration to v9

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @tecnativa

### CODE REVIEW

File: account_banking_mandate/migrations/9.0.0.2.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    # Copy mandate_id to account_move_line
+    sql = """
+    UPDATE account_move_line aml SET mandate_id = (
```
Comment (pedrobaeza) [inline account_banking_mandate/migrations/9.0.0.2.0/post-migration.py]: Please use compact form that saves one query:
```
UPDATE account_move_line aml 
SET mandate_id = ai.mandate_id
FROM account_invoice ai
WHERE aml.invoice_id=ai.id 
AND aml.invoice_id is not null
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Version folder should be 9.0.1.0.0

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done.
ups... version

File: account_banking_mandate/migrations/9.0.1.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    # Copy mandate_id to account_move_line
+    sql = """
+    UPDATE account_move_line aml
+    SET mandate_id = ai.mandate_id
+    FROM account_invoice ai
+    WHERE aml.invoice_id=ai.id
+    AND aml.invoice_id is not null;
```
Comment (pedrobaeza) [inline account_banking_mandate/migrations/9.0.1.0.0/post-migration.py]: I'm thinking that you should also add `AND aml.mandate_id IS NOT NULL` for avoiding extra operations.

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**